scripts/clean_shakespear.py

```diff
@@ -0,0 +1,19 @@
+import requests
```

Comment:
I'll be putting up a PR that introduces [pipenv](https://pipenv.readthedocs.io/en/latest/) a bit later tonight; you'll have to add `requests` to the Pipfile that that uses once that happens.

Reply:
Oh right, ended up directly going with `request`. Forgot to let you know.

Reply:
Yep, no problem! If you want to go ahead and merge this I can actually just make that change later on.